EKF/estimator_interface.cpp

```diff
@@ -366,31 +366,28 @@ void EstimatorInterface::setOpticalFlowData(uint64_t time_usec, flow_message *fl
 		// of min arrival interval because too much data is being lost
 		float delta_time = 1e-6f * (float)flow->dt;
 		float delta_time_min = 5e-7f * (float)_min_obs_interval_us;
-		bool delta_time_good = delta_time >= delta_time_min;
+		float delta_time_max = 0.1f;
```

Comment:
Is this value based on testing or set above what current supported sensors output. It would prevent use of a 10Hz sensor.

Reply:
> Is this value based on testing or set above what current supported sensors output. It would prevent use of a 10Hz sensor.

@mhkabir ?

Reply:
I set 10Hz fairly arbitrarily. I can increase to 5Hz minimum. Will this be fine? 

The PMW3901 runs at 40Hz. I'm not sure about PX4Flow.